### DISCUSSION TITLE
Is there a Static Server library that works on C++/WinRT?

### DISCUSSION THREAD

**Discussion by exotexot** [microsoft/react-native-windows#10573] [Q&A] [answered]
Hello,

In my production app I'm using a c# version of https://github.com/futurepress/react-native-static-server
I was wondering if there are any c++ modules that you known can work in a possible c++ static server module.
I don't have much experience with c++ programming, I just wanna ask the community if there is a minimal http server, which only serves some static files.

**Comment by asklar** [comment, marked as answer]
you could translate this example and the corresponding repo to C++/WinRT: https://www.codeproject.com/Articles/137979/Simple-HTTP-Server-in-C - it should be fairly straightforward once you familiarize with how to map C# types to C++/WinRT types :)
